src/main/java/mate/academy/Main.java

```diff
@@ -13,11 +12,9 @@
         manager.registerListener(sampleListener);
         manager.registerListener(loggingListener);
 
-        // Simulating an event
         Event event = new Event("UserLogin", "User123");
```

Comment:
The 'Event' class is not being imported. Verify that you are importing it correctly.